nntrainer/tensor/blas_interface.cpp

```diff
@@ -280,6 +280,25 @@ static void copy_int8_to_fp16(const unsigned int N, const uint8_t *X,
 #endif
 }
 
+static void copy_int8_to_fp16(const unsigned int N, const int8_t *X,
+                              const int incX, _FP16 *Y, const int incY) {
+  unsigned int incy = abs(incY);
+  unsigned int incx = abs(incX);
+
+#if (defined USE__FP16 && USE_NEON)
+  if (incX == 1 && incY == 1) {
+    nntrainer::neon::copy_int8_to_fp16(N, X, Y);
+  } else {
+    throw std::invalid_argument(
+      "Error: incX == 1 && incY == 1 is supported only");
```

Comment:
Why not 
```
  for (unsigned int idx = 0; idx < N; idx++) {
    Y[idx] = X[idx];
  }
```
(with performance warning...) ?

Reply:
I believe incX and incY indicate x increment and y increment. Would having an increment greater than 1 for a copy be valid?  @skykongkong8
```
  for (unsigned int idx = 0; idx < N; idx++) {
    Y[idx * incY] = X[idx * incX];
  }
```

Reply:
If the suggested code is not valid, you need to take care of LINE 296-298.

Reply:
looks like copying floating points considers incX and incY.

https://github.com/nnstreamer/nntrainer/blob/18635c269cecf0a2f85b396750e2747abf26511d/nntrainer/tensor/blas_interface.cpp#L662-L669